### PR TITLE
Shapefile to geojson

### PR DESCRIPTION
**Pull request by assafchamoy**

### CODE REVIEW

File: src/discrete-layer/components/layer-details/field-value-presentors/json.value-presentor.tsx [hunk 1]
```diff
@@ -1,87 +1,235 @@
...
+            });
+          }}
+        >
+          {btnText}
+        </Button>
+      </Box>
+    );
+  };
+
+  if (
```
Comment (alebinson) [inline src/discrete-layer/components/layer-details/field-value-presentors/json.value-presentor.tsx:179]: SURE?

**Review by alebinson — APPROVED** [review]
gr8!

File: src/discrete-layer/components/layer-details/field-value-presentors/json.value-presentor.tsx (hunk 1 above)
Comment (assafchamoy) [reply src/discrete-layer/components/layer-details/field-value-presentors/json.value-presentor.tsx:179]: If you mean the auto-focus, I did it to "enforce" validations for the imported json.. Do you think this is unnecessary?

**Comment by assafchamoy** [comment]
There was a bug with the types of the [shapefile-js](https://github.com/calvinmetcalf/shapefile-js) library, one of the method's parameters should be optional. I opened a PR for the library and maybe that will be fixed and my fix could be removed.
https://github.com/DefinitelyTyped/DefinitelyTyped/pull/62872

In the meantime, I checked what the default projection should be, and use it as a constant.

File: src/discrete-layer/components/layer-details/field-value-presentors/json.value-presentor.tsx (hunk 1 above)
Comment (assafchamoy) [reply src/discrete-layer/components/layer-details/field-value-presentors/json.value-presentor.tsx:180]: Yeah its ok

**Pull request merged**